milestones/models.py

```diff
@@ -94,7 +98,7 @@ class Meta:
         unique_together = (("course_id", "milestone"),)
 
     def __unicode__(self):
-        return unicode("%s:%s:%s" % (self.course_id, self.milestone_relationship_type, self.milestone))
+        return "%s:%s:%s" % (self.course_id, self.milestone_relationship_type, self.milestone)
```

Comment:
I think these last 3 models should also use the decorator to implement `__unicode__` and `__str__` correctly.

Reply:
With the unicode literals I think this should be fine won't they?  I'll apply the change for consistency but want to understand if you think there is a breakage I don't see.

Reply:
@jmbowman update made, please have another look.

Reply:
My concern was that `__str__` was missing under both python versions (or maybe using a less appropriate inherited implementation), and `__unicode__` is superfluous under Python 3.